### PR TITLE
Add IN-12B Nixie Tubes

### PR DESCRIPTION
**Pull request by dbrgn**
##### SUMMARY

Add IN-12B Nixie Tubes.

image

Note: Initially I was confused about the relation between Sovtek and Reflektor. According to ChatGPT (I plausibilized the statement), Sovtek is the brand and Reflektor is the production plant (owned by the USSR).

screenshot-20241223-013235

##### DATASHEETS / REFERENCES

- https://www.tube-tester.com/sites/nixie/data/in-12b.htm
- https://www.tube-tester.com/sites/nixie/dat_arch/IN-12A_IN-12B_01.pdf

##### OPEN QUESTIONS / UNRESOLVED ISSUES

n/a

##### CHECKLIST

- [x] I have read and followed the library conventions¹.
- [x] For packages, I followed IPC7351C (see details in library conventions).
- [ ] For modified elements, I bumped their version number².
- [x] I already used the new library elements in a design and verified their correctness (optional).
- [x] I'm the copyright owner of the added content (i.e. the changes are made by myself, not copied/imported from somewhere else).
- [x] I agree to publish all my changes under the CC0 Public Domain License³, allowing everyone to use and modify the content without any restrictions.

¹ *Library Conventions: https://docs.librepcb.org/#libraryconventions*
² *Minor version bump if only metadata was modified (e.g. "0.1" -> "0.1.1"), major version bump if functional changes were made (e.g. "0.1" -> "0.2")*
³ *CC0 Public Domain License: https://en.wikipedia.org/wiki/CC0*

### CODE REVIEW

File: pkg/d06de9dc-d4ab-4de4-ac44-b544e4ff80bb/package.lp [hunk 1]
```diff
@@ -0,0 +1,204 @@
+(librepcb_package d06de9dc-d4ab-4de4-ac44-b544e4ff80bb
+ (name "Sovtek IN-12")
```
Comment (ubruhin) [inline pkg/d06de9dc-d4ab-4de4-ac44-b544e4ff80bb/package.lp]: Package naming convention:
```suggestion
 (name "SOVTEK_IN-12")
```

**Review by ubruhin — COMMENTED** [review]
Theoretically the documentation polygon is missing and the legend polygon should be *outside* the package outline (currently both polygons are identical):

image

But since it's more a "fun" package anyway, feel free to merge as-is :wink:

**Comment by dbrgn** [comment]
> Theoretically the documentation polygon is missing

Oops, you're right!

> and the legend polygon should be outside the package outline (currently both polygons are identical):

I originally kept them like that because every tube has slightly different dimensions, so it won't be exact anyways. But since I edited the outlines to add the documentation layer, I also fixed that one.

File: pkg/d06de9dc-d4ab-4de4-ac44-b544e4ff80bb/package.lp (hunk 1 above)
Comment (dbrgn) [reply pkg/d06de9dc-d4ab-4de4-ac44-b544e4ff80bb/package.lp]: I had it like that initially, but then I re-read the package naming convention 😄

screenshot-20241224-230546

We should probably adjust that, or maybe add a dedicated item about manufacturer-specific naming.

Comment (ubruhin) [reply pkg/d06de9dc-d4ab-4de4-ac44-b544e4ff80bb/package.lp]: Huh, I was not aware the conventions are outdated :scream:

Fixed in https://github.com/LibrePCB/librepcb-doc/pull/78.

**Review by ubruhin — APPROVED** [review]

**Pull request merged**